prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -39,16 +39,6 @@
 #include "rxr_pkt_cmd.h"
 #include "rxr_read.h"
 
-/*
- * Utility constants and funnctions shared by all REQ packe
- * types.
- */
-struct rxr_req_inf {
-	uint64_t protover;
-	uint64_t base_hdr_size;
-	uint64_t ex_feature_flag;
-};
-
```

Comment:
We do not need this change any more.

Reply:
I do not think so. This change is independent of queuing logic. Both `RXR_REQ_FEATURE_DELIVERY_COMPLETE` and `RXR_REQ_FEATURE_RDMA_READ` are extra features.

Reply:
Change reverted. Resolving.